docs/guides/unbound.md

```diff
@@ -8,22 +8,22 @@ When you operate your own (tiny) recursive DNS server, then the likeliness of ge
 
 ### What *is* a recursive DNS server?
 
-The first distinction we have to be aware of is whether a DNS server is *authoritative* or not.  If I'm the authoritative server for, e.g., `pi-hole.net`, then I know which IP is the correct answer for a query. Recursive name servers, in contrast, resolve any query they receive by consulting the servers authoritative for this query by traversing the domain.
+The first distinction we have to be aware of is whether a DNS server is *authoritative* or not. If I'm the authoritative server for, e.g., `pi-hole.net`, then I know which IP is the correct answer for a query. Recursive name servers, in contrast, resolve any query they receive by consulting the servers authoritative for this query by traversing the domain.
 Example: We want to resolve `pi-hole.net`. On behalf of the client, the recursive DNS server will traverse the path of the domain across the Internet to deliver the answer to the question.
 
 ### What does this guide provide?
 
 In only a few simple steps, we will describe how to set up your own recursive DNS server. It will run on the same device you're already using for your Pi-hole. There are no additional hardware requirements.
 
-This guide assumes a fairly recent Debian/Ubuntu based system and will use the maintainer provided packages for installation to make it an incredibly simple process. It assumes only very basic knowledge of how DNS works.
+This guide assumes a fairly recent Debian/Ubuntu-based system and will use the maintainer provided packages for installation to make it an incredibly simple process. It assumes only a very basic knowledge of how DNS works.
```

Comment:
Maybe this should read `... of how a DNS works` or `...of how DNS work`?

Reply:
Depends on if you take DNS to mean Domain Name Server or the traditional Domain Name System.

Cloudflare documentation uses "How does DNS work?"

Reply:
Let's go with "how DNS work". It feels more natural

Reply:
Sounds good.